ouroboros-consensus/test-consensus/Test/Dynamic/General.hs

```diff
@@ -220,3 +262,53 @@ prop_general k TestConfig{numSlots, nodeJoinPlan, nodeTopology} schedule
       where
         NumSlots t = numSlots
         NodeJoinPlan m = nodeJoinPlan
+
+    -- check for Chain Growth violations if there are no Common Prefix
+    -- violations
```

Comment:
Can you add a link to the paper + section/page where this is defined? That would be handy.

Reply:
Done; quoted the property too.